there/repl_connection.py

```diff
@@ -260,11 +261,11 @@ def write_to_file(self, path, contents):
         """
         if not isinstance(contents, (bytes, bytearray)):
             raise TypeError('contents must be bytes/bytearray, got {} instead'.format(type(contents)))
-        blocksize = 128
+        blocksize = 512
         self.exec('_f = open({!r}, "wb")'.format(str(path)))
         for i in range(0, len(contents), blocksize):
-            self.exec('_f.write({!r})'.format(contents[i:i+blocksize]))
-        self.exec('_f.close(); del _f;')
+            self.exec('_f.write({!r})'.format(contents[i:i+blocksize]), timeout=60)
```

Comment:
You're right. It works without this "timeout=60". I will remove it.
An overall timeout is not necessary, because writing a block is either quick or fails. And I think, if it fails then exec() will raise IOError and the for-loop will end immediately.
Do you agree?